packages/messaging/android.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @module Android Messaging
```

Comment:
This is the core of the change. I've moved the messaging logic here to mimic what we have for Webkit. It's much simpler overall. I think we can use this for the basis of the C-S-S migration.

Reply:
yes this is a really good idea, thanks.